src/ApiClient/ServerClient.ts

```diff
@@ -1,4 +1,4 @@
-import { AxiosError, AxiosInstance, AxiosRequestConfig, AxiosResponse } from "axios";
+import { AxiosError, AxiosRequestConfig, AxiosResponse } from "axios";
```

Comment:
I can't put a review comment since its outside the diff, but ServerClient#loggingIn is unused and making my IDE complain